### PR TITLE
server: Add to README a troubleshooting for xsnd

### PR DESCRIPTION
**Pull request by JonSalazar**
### Problem

Following the README steps to get running the server, sometimes it happens not mentioned issues.

### Solution

It was added troubleshooting to server/README for mentioned issues.

### Result

server/README file has troubleshooting for xsnd steps.

### CODE REVIEW

File: server/README.md [hunk 1]
```diff
@@ -11,6 +11,25 @@
 
 3. Run the application with: `sbt run`
 
+## Troubleshooting
+
+1. If when you run xsnd you get
```
Comment (AlexITC) [inline server/README.md]: If you get the following error while running the explorer: `error: Could not locate RPC credentials.`, then, create a file at`~/.xsncore/xsn.conf` with this content:
    ```
    rpcuser=dummy
    rpcpassword=replaceme
    rpcport=51473
    txindex=1
    ```

Now, `xsnd` can be started without arguments, for example, `./bin/xsnd`.

File: server/README.md [hunk 2]
```diff
@@ -11,6 +11,25 @@
...
+    create a file `~/.xsncore/xsn.conf` with
+    ```
+    rpcuser=dummy
+    rpcpassword=replaceme
+    rpcport=51473
+    txindex=1
+    ```
+    after this step, you can run it with just `bin/xsnd`
+
+2. If you have connection problems with xsnd node, try disabling IPv6.
```
Comment (AlexITC) [inline server/README.md]: If the `xsnd` node is not getting any blocks/peers, try disabling IPv6, and remove the peers file (`~/.xsncore/peers.dat`), restarting `xsnd` should get peers now.

**Review by AlexITC — CHANGES_REQUESTED** [review]

File: server/README.md (hunk 2 above)
Comment (JonSalazar) [reply server/README.md]: Ok

File: server/README.md (hunk 1 above)
Comment (JonSalazar) [reply server/README.md]: Ok

File: server/README.md [hunk 3]
```diff
@@ -11,6 +11,15 @@
 
 3. Run the application with: `sbt run`
 
+## Troubleshooting
+
+1. If you get the following error while running the explorer: `error: Could not locate RPC credentials.`, then, create a file at `~/.xsncore/xsn.conf` with this content:
+`rpcuser=dummy rpcpassword=replaceme rpcport=51473 txindex=1`. Now, xsnd can be started without arguments, for example, `./bin/xsnd`.
```
Comment (AlexITC) [inline server/README.md]: Now this is a corrupted xsn file config.

Comment (JonSalazar) [reply server/README.md]: why is that? is necessary to remove all the content in .xsncore before the update?

**Review by AlexITC — APPROVED** [review]

**Pull request merged**